js/main.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener("DOMContentLoaded", function() {
+  fillRegionSelect(regions);
+});
+document.querySelector("form").addEventListener("change", validate);
+const submit = document.querySelector('#submit');
+const checkbox = document.querySelector('#checkbox');
+const option = (...args) => new Option(...args);
+
+function validate(ev) {
+  let target = ev.target;
+  let validName = isValid(document.querySelector('#name'), regExp.name);
```

Comment:
boolean variable names typically start with is/has/etc. When you read the name it should be clear it's a boolean.